tests/guardrails/suspend_threshold.tests.js

```diff
@@ -0,0 +1,60 @@
+const { limits } = require('../../src/index');
+
+var suspendThresholdTests = (test) => {
+    it('can set max units with warning', async () => {
+        await test.settingsScreen.SetSuspendThreshold(
```

Comment:
I believe the suspend threshold is on a separate screen, so we can mirror all of the other tests with an `OpenSuspendThresholdScreen`, do all of the tests and a `Close`?